modules/las/src/lib/parse-las.ts

```diff
@@ -151,8 +167,18 @@ export function parseLASChunked(rawData, skip, onParseData) {
   }
 }
 
-function detectTwoByteColors(colorDepth, decoder, batchSize) {
-  let twoByteColor;
+/**
+ * @param decoder
+ * @param batchSize
+ * @param colorDepth
+ * @returns boolean
+ */
+function detectTwoByteColors(
+  decoder: any = {},
+  batchSize: number,
+  colorDepth?: number | string
+): boolean {
+  let twoByteColor: any | boolean;
```

Comment:
What's the point of using `any | bool` here? Maybe do `twoByteColor` `false` or `true` by default and make it bool type only?

Reply:
> What's the point of using `any | bool` here? Maybe do `twoByteColor` `false` or `true` by default and make it bool type only?

ok